front/lib/api/files/upload.ts

```diff
@@ -291,121 +280,91 @@ type ProcessingFunction = (
   file: FileResource
 ) => Promise<Result<undefined, Error>>;
 
-type ProcessingPerUseCase = {
-  [k in FileUseCase]: ProcessingFunction | undefined;
-};
+const getProcessingFunction = ({
```

Comment:
Refactored as a function (instead of the mapping table) as it was really easy to forget to update all mimes types doing the same processing and the mapping table was getting very big.